functions/firebase/FirebaseFirestore/Function.cs

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// [START functions_cloudevent_firebase_firestore]
 // [START functions_firebase_firestore]
```

Comment:
Will you update all the docs to reference this sample by the new region tag?
Once you've done that, we should remove the old region tag. Let me know and I'll create an issue so we don't forget about that.

Reply:
We can leave the old tag for gen 1 docs and use the new one for gen 2 docs which reference all samples from all languages with the new tag.

Reply:
SGTM